src/lib/utils/sizes.js

```diff
@@ -22,7 +22,7 @@ const DESIGN_WIDTH = 360
  * @param {boolean} isMax: should or shouldnt use Math.min
  */
 export const getDesignRelativeWidth = (width, isMax = true) => {
-  const screenWidth = isPortrait() ? getScreenWidth() : getScreenHeight()
+  const screenWidth = isPortrait() ? getMaxDeviceWidth() : getMaxDeviceHeight()
```

Comment:
have you checked what's happened if resize browser window lesser that `maxWidthForTabletAndDesktop` theme value ? Would it be responsive in this case ? if yes - we have  to take in into account

Reply:
everything wiil work as before, the only thing changed is that we restrict max app width with 'maxWidthForTabletAndDesktop' value for desctops and tablets.

Reply:
ok i remember i found during testing Zoom that there're no adaptive design - if make window smaller (e.g. by opening the console) - scrollbars will appears

so ok, we've took this case into the account